pkg/repository/aws/profileDynamoManager.go

```diff
@@ -0,0 +1,110 @@
+package aws
```

Comment:
Can/Should we move some of the general dynamo functionality to [this file](https://github.com/CollActionteam/collaction_backend/blob/development/pkg/repository/aws/dynamo.go=?